docker/run_es

```diff
@@ -1 +1 @@
-docker run -d --net curation -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xmx4g -Xms4g" --name elasticsearch docker.elastic.co/elasticsearch/elasticsearch:7.10.2
+docker run -d --net curation -p 9200:9200 -p 9300:9300 -e "discovery.type=single-node" -e "ES_JAVA_OPTS=-Xmx8g -Xms8g" --name elasticsearch docker.elastic.co/elasticsearch/elasticsearch:7.10.2
```

Comment:
Why does elasticSearch suddenly need 8GB? I don't think ES should be impacted by the authentication fix?

Reply:
this was local dev stuff to get the index all working without errors I can leave it out of this PR but this file is for local devs